### DISCUSSION TITLE
I didnt get the educational benefits within 5 days of being approved

### DISCUSSION THREAD

**Discussion by souptik-samanta** [community/community#135954] [GitHub Education] [answered]
### Hi everyone,

can you guys assist me with how to see the educational benefits or where to get all
and how to use the jet brains ide

**Comment by davevad93** [comment, marked as answer]
Hi @souptik-samanta ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @souptik-samanta was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145, #135407

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
